.pre-commit-config.yaml

```diff
@@ -19,10 +19,8 @@ repos:
     hooks:
       - name: Prettier
         id: prettier
-  - repo: local
+  - repo: https://github.com/golangci/golangci-lint
```

Comment:
I just recently changed this so it would use the version we either have installed via asdf or nix and we don't have to keep yet another version up to date here.